requirements_dev.txt

```diff
@@ -1,14 +1,18 @@
-pip==9.0.1
-bumpversion==0.5.3
-wheel==0.30.0
-watchdog==0.8.3
-flake8==3.5.0
-tox==2.9.1
-coverage==4.5.1
-Sphinx==1.7.0
-sphinx_rtd_theme==0.2.4
-
-pytest==3.4.2
-pytest-runner==2.11.1
+-r requirements.txt
+
+pip>=9.0.1,<10.0
```

Comment:
do you generate the list with some tool? I would personally keep the list without versions, and use frozen version only if required for compatibility reasons.